packages/core/src/types.ts

```diff
@@ -30,35 +30,30 @@ type GriffelStylesCSSPseudos = {
 // updated or removed.
 //
 
-type GriffelStylesCSSObjectCustomL1 =
-  | ({
-      [Property: string]: string | undefined | GriffelStylesCSSObjectCustomL2;
-    } & Partial<GriffelStylesUnsupportedCSSProperties>)
```

Comment:
Removed `Partial<GriffelStylesUnsupportedCSSProperties>` as that is already a part of `GriffelStylesStrictCSSObject`